benchmarks/src/bin/h2o.rs

```diff
@@ -26,7 +26,7 @@ use datafusion::datasource::listing::{
 use datafusion::datasource::MemTable;
 use datafusion::prelude::CsvReadOptions;
 use datafusion::{arrow::util::pretty, error::Result, prelude::SessionContext};
-use datafusion_benchmarks::BenchmarkRun;
+use datafusion_benchmarks::util::BenchmarkRun;
```

Comment:
i think the real problem (if any) is that `datafusion_benchmarks::BenchmarkRun` and `datafusion_benchmarks::util::BenchmarkRun` are both valid import path.

there are 3 ways to go about it

- stop re-exporting BenchmarkRun from crate root, require `::util::`
- make util private, accept only BenchmarkRun import from crate root
- ignore the problem and give up on consistency

Reply:
> stop re-exporting BenchmarkRun from crate root, require ::util::

I think that is what this PR does which seems reasonable to me